src/main/java/christmas/model/EventDay.java

```diff
@@ -0,0 +1,57 @@
+package christmas.model;
+
+
+import christmas.model.exception.DateException;
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.Month;
+
+public class EventDay {
+    private static final int CHRISTMAS_DATE = 25;
+    private static final int YEAR = 2023;
+    private static final int START_DAY = 1;
+    private static final int END_DAY = 31;
+    private final int day;
+
+    public EventDay(String inputDay) {
+        int day = convertDay(inputDay);
+        validateDateRange(day);
+        this.day = day;
+    }
+
+    private void validateDateRange(int date) {
+        if (date < START_DAY || date > END_DAY) {
+            throw new DateException();
+        }
+    }
+
+    private int convertDay(String inputDay) {
+        try {
+            return Integer.parseInt(inputDay);
+        } catch (NumberFormatException exception) {
+            throw new DateException();
+        }
+    }
+
+    public boolean isBeforeChristmas() {
+        return day <= 25;
+    }
+
+    public boolean isWeekend() {
+        DayOfWeek dayOfWeek = getLocalDate();
+        return dayOfWeek.equals(DayOfWeek.FRIDAY) || dayOfWeek.equals(DayOfWeek.SATURDAY);
+    }
+
+    public boolean isSpecialDay() {
+        return getLocalDate().equals(DayOfWeek.SUNDAY) || day == CHRISTMAS_DATE;
```

Comment:
```suggestion
        return getLocalDate == DayOfWeek.SUNDAY || day == CHRISTMAS_DATE;
```
`DayOfWeek` 타입은 enum이라서 `==`으로 비교해도 됩니다!
실제로 enum의 equals의 내부 구현을 들여다보면 `==`으로 구현되어 있습니다